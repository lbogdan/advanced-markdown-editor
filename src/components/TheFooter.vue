<template>
  <v-footer
    app
    absolute
    padless
  >
    <v-spacer />
    <v-col class="text-center v-footer--padless">
      Copyright &copy; {{ new Date().getFullYear() }}
      <a
        class="white--text"
        href="https://github.com/paul-soporan"
      >
        <strong>Paul Soporan</strong>
      </a>
    </v-col>
    <v-col class="text-center v-footer--padless">
      Released under the
      <a
        class="white--text"
        href="https://github.com/paul-soporan/advanced-markdown-editor/blob/master/LICENSE.md"
      >
        <strong>MIT License</strong>
      </a>
    </v-col>
    <v-spacer />
  </v-footer>
</template>


<script lang="ts">

import Vue from 'vue';
import Component from 'vue-class-component';

@Component
export default class TheFooter extends Vue {
}

</script>
