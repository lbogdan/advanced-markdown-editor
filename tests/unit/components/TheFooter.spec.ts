import Vue from 'vue';
import Vuetify from 'vuetify';

import {
  shallowMount,
  createLocalVue,
} from '@vue/test-utils';

import TheFooter from '@/components/TheFooter.vue';


Vue.use(Vuetify);

const localVue = createLocalVue();


describe('TheFooter', () => {
  let wrapper = shallowMount(TheFooter);

  beforeEach(() => {
    wrapper = shallowMount(TheFooter, {
      localVue,
    });
  });

  it('matches snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
