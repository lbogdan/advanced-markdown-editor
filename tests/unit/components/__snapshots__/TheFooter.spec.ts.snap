// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter matches snapshot 1`] = `
<v-footer-stub height="auto" tag="div" tile="true" absolute="true" app="true" padless="true">
  <v-spacer-stub></v-spacer-stub>
  <v-col-stub tag="div" class="text-center v-footer--padless">
    Copyright Â© 2019
    <a href="https://github.com/paul-soporan" class="white--text"><strong>Paul Soporan</strong></a></v-col-stub>
  <v-col-stub tag="div" class="text-center v-footer--padless">
    Released under the
    <a href="https://github.com/paul-soporan/advanced-markdown-editor/blob/master/LICENSE" class="white--text"><strong>MIT License</strong></a></v-col-stub>
  <v-spacer-stub></v-spacer-stub>
</v-footer-stub>
`;
